conan/tools/gnu/makedeps.py

```diff
@@ -651,7 +670,7 @@ def generate(self) -> None:
             if require.build:
                 continue
 
-            dep_gen = DepGenerator(dep, require)
+            dep_gen = DepGenerator(dep, require, self._conanfile.output)
```

Comment:
```suggestion
            output = ConanOutput(scope=f"{self._conanfile} MakeDeps: {dep}:")
            dep_gen = DepGenerator(dep, require, output)
```
I'd say the warnings might be a bit limited in the information scope, not showing it is a warning from ``MakeDeps`` generator, and about one specific dep property.

It might be necessary to add an ``import``, this is mostly a rough idea.

Reply:
Done.